eng/Packages.Data.props

```diff
@@ -105,6 +105,7 @@
     <PackageReference Update="Microsoft.Identity.Client" Version="4.39.0" />
     <PackageReference Update="Microsoft.Identity.Client.Desktop" Version="4.39.0" />
     <PackageReference Update="Microsoft.Identity.Client.Extensions.Msal" Version="2.19.3" />
+    <PackageReference Update="Microsoft.Graph" Version="4.28.0" />
```

Comment:
Do we need to add this line here? Considering that we only use this package in KeyVault, I think it's better to only include this package in `Azure.ResourceManager.KeyVault.Tests.csproj`.